tests/resources/Docker-Util.robot

```diff
@@ -51,6 +53,44 @@ Get Container IP
     Should Be Equal As Integers  ${rc}  0
     [Return]  ${ip}
 
+# Use host docker daemon to start a dind container alongside this test container
+Start Docker Daemon On TCP
+    [Arguments]  ${docker-params}  ${dind-version}=docker:1.13-dind
+    ${rc}  ${id}=  Run And Return Rc And Output  docker run -it --privileged ${docker-params} -d ${dind-version}
+    Should Be Equal As Integers  ${rc}  0
+    [Return]  ${id}
+
+# The local dind version is embedded in Dockerfile
+# docker:1.13-dind
+# If you are running this keyword in a container, make sure it is run with --privileged turned on
+Start Docker Daemon Locally
+    [Arguments]  ${dockerd-params}  ${dockerd-path}=/usr/local/bin/dockerd-entrypoint.sh  ${log}=/var/log/daemon-local.log
```

Comment:
for the CI at least, we don't grab /var/log logs, default might better be set to the local folder?